### PR TITLE
fix(grpc): logging error messages

### PR DESCRIPTION
**Pull request by renatoaguimaraes**
closes [PLT-115]

[PLT-115]: https://ydata.atlassian.net/browse/PLT-115?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pkg/common/grpc/logging_interceptor.go [hunk 1]
```diff
@@ -10,12 +10,12 @@ import (
...
-		logger.Infof("gRPC req %s -> %v", info.FullMethod, req)
+		logger.Infof("gRPC req %s -> [%v]", info.FullMethod, req)
 		res, err := handler(ctx, req)
-		if err == nil {
-			logger.Errorf("gRPC err %s -> %v", info.FullMethod, err)
+		if err != nil {
+			logger.Errorf("gRPC err %s -> [%v]", info.FullMethod, err)
 		}
-		logger.Infof("gRPC res %s -> %v", info.FullMethod, res)
+		logger.Infof("gRPC res %s -> [%v]", info.FullMethod, res)
```
Comment (portellaa) [inline pkg/common/grpc/logging_interceptor.go:18]: can we give some spaces in here? 😋

**Review by portellaa — COMMENTED** [review]
ups 😅  missed that

File: pkg/common/grpc/logging_interceptor.go (hunk 1 above)
Comment (renatoaguimaraes) [reply pkg/common/grpc/logging_interceptor.go:18]: do you mean between square brackets?

Comment (portellaa) [reply pkg/common/grpc/logging_interceptor.go:18]: like this:

```go
...
res, err := handler(ctx, req)

if err != nil {
  logger.Errorf("gRPC err %s -> [%v]", info.FullMethod, err)
}

logger.Infof("gRPC res %s -> [%v]", info.FullMethod, res)
```

**Comment by renatoaguimaraes** [comment]
> ups 😅 missed that

me too

File: pkg/common/grpc/logging_interceptor.go (hunk 1 above)
Comment (portellaa) [reply pkg/common/grpc/logging_interceptor.go]: well, at least i prefer, don't know. what do you think? we should define a guideline

Comment (renatoaguimaraes) [reply pkg/common/grpc/logging_interceptor.go]: sounds good to me

**Review by portellaa — APPROVED** [review]

**Pull request merged**